googletest/test/googletest-death-test-test.cc

```diff
@@ -1406,6 +1407,15 @@ TEST(MatcherDeathTest, PolymorphicMatcherDoesNotMatch) {
       "Expected: contains regular expression \"rest is science\"");
 }
 
+// Shows that death tests are fast when core dumps can be disabled.
```

Comment:
Please remove this test since the only way for it to fail is by timeout. Automated testing will not catch a failure here.